docs/_tooling/extensions/score_draw_uml_funcs/helpers.py

```diff
@@ -10,65 +10,132 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 # *******************************************************************************
-from itertools import chain
+import re
 
 
 def gen_alias(title: str) -> str:
     return "".join(word[0] for word in title.split())
 
 
-def gen_link_text(alias_from: str, alias_to: list[str], link_text: str) -> str:
+def gen_format(need: dict) -> str:
+    if need["safety"] == "ASIL_B":
+        color = "<<asilb>>" if "comp_arc_sta" in need["type"] else ""
+    else:
+        color = ""
+
+    return color
```

Comment:
```suggestion
    if need["safety"] == "ASIL_B":
       return "<<asilb>>" if "comp_arc_sta" in need["type"] else ""
    return ""
```

@AlexanderLanin  unsure if we want to have early returns or not to make some logic more condenst?